src/Symfony/Installer/NewCommand.php

```diff
@@ -356,6 +358,30 @@ private function cleanUp()
     {
         $this->fs->remove(dirname($this->compressedFilePath));
 
+        try {
+            $this->fs->remove(array(
+                $this->projectDir.'/LICENSE',
+                $this->projectDir.'/UPGRADE.md',
+                $this->projectDir.'/UPGRADE-2.2.md',
+                $this->projectDir.'/UPGRADE-2.3.md',
+                $this->projectDir.'/UPGRADE-2.4.md',
+                $this->projectDir.'/UPGRADE-2.5.md',
+                $this->projectDir.'/UPGRADE-2.6.md',
+                $this->projectDir.'/UPGRADE-2.7.md',
+                $this->projectDir.'/UPGRADE-3.0.md',
```

Comment:
This list should be build dynamically with a glob, to remove any ``UPGRADE-*.md`` file. This would avoid having to update the installer for each minor version to add a new file. And what about the changelog